assets/src/data/model/ticket/index.js

```diff
@@ -91,24 +93,21 @@ export const whereConditions = ( {
 	month = 'none',
 } ) => {
 	const where = [];
-	const GREATER_AND_EQUAL = encodeURIComponent( '>=' );
-	const LESS_AND_EQUAL = encodeURIComponent( '<=' );
-
 	if ( ! showExpired ) {
 		where.push(
-			'where[TKT_end_date**expired][]=>' +
+			'where[TKT_end_date**expired][]=' + GREATER_THAN +
 			'&where[TKT_end_date**expired][]=' +
 			nowDateAndTime.local().format()
 		);
 	}
 	if ( month && month !== 'none' ) {
 		where.push(
-			'where[TKT_start_date][]=' + GREATER_AND_EQUAL +
+			'where[TKT_start_date][]=' + GREATER_THAN +
 			'&where[TKT_start_date][]=' +
 			moment().month( month ).startOf( 'month' ).local().format()
 		);
 		where.push(
-			'where[TKT_end_date][]=' + LESS_AND_EQUAL +
+			'where[TKT_end_date][]=' + LESS_THAN +
 			'&where[TKT_end_date][]=' +
```

Comment:
as noted earlier, you need to retain `GREATER_AND_EQUAL` and `LESS_AND_EQUAL` instead of changing those.  The only thing needing implemented is the `GREATER_THAN` to replace the `>` in the `showExpired` condition.  For months, we must include the starting date and the ending date in the query otherwise a query for the month of June would exclude events falling on June 1 or June 30.